src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/OperationNameTelemetryInitializer.cs

```diff
@@ -15,7 +15,7 @@
 
     public class OperationNameTelemetryInitializer : TelemetryInitializerBase
     {
-        public const string BeforeActionNotificationName = "Microsoft.AspNet.Mvc.BeforeAction";
+        public const string BeforeActionNotificationName = "Microsoft.AspNetCore.Mvc.BeforeAction";
```

Comment:
Not aware of any deliberate change here. @rynowak any thoughts?

---
In reply to: [60813471](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/164#discussion_r60813471) [](ancestors = 60813471)